Report/Chapters/Testplan.tex

```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
 The acceptance tests are focused on the front end.
 
 \section{Acceptance tests} \label{acceptance tests}
+	\subsection{Test 1}
+		This test evaluates the functioning of the FunctionMenu and the spawning of basic blocks.
+		
+		\begin{enumerate}
+			\item Right click on the workspace to open the FunctionMenu.
+			\item Click on the item labelled Basic.
+			\item Observe that contents are present in the opened list.
+			\item Click on the Basic item again to close the list.
+			\item Press the button at the bottom of the menu labelled Value Block.
+			\item Enter "Hello" into the opened dialog window.
```

Comment:
Something like this?

    Enter "Hello" into the opened dialog window (including quotation marks).